src/app/(proper_react)/(redesign)/(authenticated)/admin/feature-flags/components/DeleteFeatureFlag.tsx

```diff
@@ -0,0 +1,40 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+"use client";
+
+import { useRouter } from "next/navigation";
+
+export const DeleteFeatureFlag = (props: { name: string | undefined }) => {
```

Comment:
Nit, but I assume this should always be defined?

```suggestion
export const DeleteFeatureFlag = (props: { name: string }) => {
```